MicrosoftDX/nether
# Initial iOS SDK PR

N.A. as no manual code is involved yet (except Makefiles, bash scripts and config files)

### Description:
This first iOS SDK checkin provides the machinery on MacOS to enable the creation of an Objective-C CocoaPod and all the necessary scripts to actually create such a pod as repository "netherpod" in the current GitHub account. You may add the pod to your Xcode project, it should actually work.
Please note, that this first contains the automatic part of the API (generated from Swagger). The manual parts (Event-Hub messaging, Hartbeat etc.) will follow in the next days. You'll be busy first in setting up and make this drop work first, if it does you're prepared to get the additional parts.

### Test:
- Build Nether and run it locally
- Go to the iOS SDK source directory and run "make"
- Most likely you'll first encounter errors telling you which software you are missing on your Mac. Follow the instructions und re-run make until it builds successfully.

(Make sure that all above information is provided before submitting your pull request)
